gulpfile.js

```diff
@@ -314,10 +314,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '1.12.1'))
+    .pipe(replace('[ios.current-app-version]', '1.13.0'))
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '1.12'))
+    .pipe(replace('[android.current-app-version]', '1.13.0'))
```

Comment:
```suggestion
    .pipe(replace('[android.current-app-version]', '1.13.2'))
```

Should be 1.13.2 (see https://github.com/corona-warn-app/cwa-app-android/releases/tag/v1.13.2)